app/(main)/markets/News.tsx

```diff
@@ -6,9 +6,10 @@ import Image from 'next/image';
 import { FA_EVENTS } from '@/analytics';
 import { Skeleton } from '@/app/components';
 import { getMarketInsights, MarketInsights } from '@/queries/gnosis-ai';
+import { FixedProductMarketMaker } from '../../../queries/omen';
```

Comment:
can we use @ syntax? 

```suggestion
import { FixedProductMarketMaker } from '@/queries/omen';
```